kafka-ui-react-app/src/components/Connect/Details/Actions/__tests__/Actions.spec.tsx

```diff
@@ -27,7 +37,33 @@ jest.mock(
 );
 
 describe('Actions', () => {
-  containerRendersView(<ActionsContainer />, Actions);
+  const actionsContainer = (props: Partial<ActionsProps> = {}) => (
+    <ThemeProvider theme={theme}>
+      <Provider store={store}>
+        <Router history={history}>
+          <ActionsContainer>
+            <Actions
+              data-testid="actions_view"
+              deleteConnector={jest.fn()}
+              isConnectorDeleting={false}
+              connectorStatus={ConnectorState.RUNNING}
+              restartConnector={jest.fn()}
+              restartTasks={jest.fn()}
+              pauseConnector={jest.fn()}
+              resumeConnector={jest.fn()}
+              isConnectorActionRunning={false}
+              {...props}
+            />
+          </ActionsContainer>
+        </Router>
+      </Provider>
+    </ThemeProvider>
+  );
+
+  it('container renders view', () => {
+    render(actionsContainer());
```

Comment:
please use `render` method from `lib/testHelpers`. this `render` already contains all these wrappers

Reply:
done